configure

```diff
@@ -4248,7 +4248,15 @@ then :
 
 fi
 
-# overlayfs features temporarily disabled pending fixes
+# The --overlay and related options are not supported anymore due to security
+# concerns (see CVE-2021-26910 and #4178), issues on newer kernels (see #2799)
+# and overall lack of interest in fixing and maintaining them.
+#
+# Support is unlikely to be restored, as it would require auditing the
+# overlayfs code to ensure that it is secure and that it works properly.
```

Comment:
Technical notes:
- crablock and bubblewrap have integrated support for overlayfs. This was possible because Linux >= 5.11 allows `overlayfs` mounts from unprivileged user-namespace. An "unprivileged" implementation in firejail might be the most likely comeback as it means less auditing.
- The usage in firejail was broken at least two times in the past because of kernel changes (IIRC). And is likely to break again in the future.
  https://www.kernel.org/doc/html/latest/filesystems/overlayfs.html
  > Changes to the underlying filesystems while part of a mounted overlay filesystem are not allowed. If the underlying filesystem is changed, the behavior of the overlay is undefined, though it will not result in a crash or deadlock.

Reply:
> Technical notes:

Thanks for the all details, they seem to be very relevant.

Though this information will likely get lost in this PR; thoughts on adding it
to #4178, the wiki or a new dedicated issue?